src/roboat_ros_node.py

```diff
@@ -157,9 +159,10 @@ def publish(self, agents_pos, pred_velocities):
                idx = ts_id * self.model_args.output_pred_state_dim * self.model_args.n_mixtures + mx_id
                pt.x = prev_x + self.model_args.dt * pred[0][idx + (self.model_args.n_mixtures if SWITCH_AXES else 0)]
                pt.y = prev_y + self.model_args.dt * pred[0][idx + (0 if SWITCH_AXES else self.model_args.n_mixtures)]
-               path_mrk.points.append(pt)
                pose.pose.position = pt
                path_msg.path.poses.append(pose)
+               pt.z = 0.2
```

Comment:
Display the points a little bit higher than z=0, then I can display them in rviz above the map. Otherwise they are not visible